### ISSUE TITLE
Style/EmptyLinesAroundArguments has a bad default: makes documentation worse

### ISSUE THREAD

**Issue by jaredbeck** [rubocop-hq/rubocop#9365]
## Is your feature request related to a problem? Please describe.

Please consider the following documentation.

```ruby
foo(
  1,

  # Documentation that only applies to 2
  # Often over multiple lines
  2,

  3
)
```

We use empty lines intentionally, to demonstrate that the documentation applies only to 2.

By forbidding the use of such empty lines, the current default makes documentation worse.

## Describe the solution you'd like

Change the default of `Style/EmptyLinesAroundArguments` to allow such documentation.

## Describe alternatives you've considered

Disable the `Style/EmptyLinesAroundArguments` cop in all my projects.

**Comment by forthrin** [comment]
Rather than putting a band-aid on the Titanic, stay clear of the iceberg.

```ruby
a = 1
c = 3
# Now you can write as much documentation as you want
# (Add LotR fan fiction here)
b = 2
foo(a, b, c)

```

Suggest closing issue.
